src/components/Claim/Link/Onchain/Success.view.tsx

```
@@ -53,6 +63,39 @@ export const SuccessClaimLinkView = ({ transactionHash, claimLinkData, type }: _
         }
     }, [isPeanutWallet, transactionHash, type])
 
+    useEffect(() => {
+        if (!!transactionHash) return
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)